content/collections.md

```diff
@@ -358,12 +358,17 @@ Migrations.add({
       batch.find({_id: list._id}).updateOne({$set: {todoCount}});
     });
 
-    // We need to wrap the async function to get a synchronous API that migrations expects
-    const execute = Meteor.wrapAsync(batch.execute, batch);
-    return execute();
+    //Mongo throws an error if we execute a batch operation without actual operations, e.g. when Lists was empty.
+    if(batch.s && batch.s.currentBatch && batch.s.currentBatch.operations && batch.s.currentBatch.operations.length > 0){
```

Comment:
I'm actually not sure the latest version of Mongo still throws an error when bulk ops are executed without any operations:

```js
rs0:PRIMARY> db.testtesttest.initializeOrderedBulkOp().execute()
BulkWriteResult({
	"writeErrors" : [ ],
	"writeConcernErrors" : [ ],
	"nInserted" : 0,
	"nUpserted" : 0,
	"nMatched" : 0,
	"nModified" : 0,
	"nRemoved" : 0,
	"upserted" : [ ]
})
```

Thoughts?